jetty-security/src/main/java/org/eclipse/jetty/security/SecurityHandler.java

```diff
@@ -76,11 +77,8 @@
 
     static
     {
-        for (Authenticator.Factory factory : ServiceLoader.load(Authenticator.Factory.class))
-        {
-            __knownAuthenticatorFactories.add(factory);
-        }
-
+        TypeUtil.serviceStream(ServiceLoader.load(Authenticator.Factory.class))
+            .forEach(__knownAuthenticatorFactories::add);
```

Comment:
Discovering `AuthenticatorFactory`s with the `ServiceLoader` is a very recent change so I think this is okay. It will also allow jetty-jaspi to work in jetty-all.